dynamiq/utils/embedding_validators.py

```diff
@@ -0,0 +1,54 @@
+from typing import Any
+
+
+def validate_embedding(embedding: Any) -> tuple[bool, str]:
```

Comment:
Such helpers are used just by embedders, so please move them to the embedders/base.py.